tests/TestCase/Database/Schema/PostgresSchemaTest.php

```diff
@@ -9,10 +9,10 @@
  * For full copyright and license information, please see the LICENSE.txt
  * Redistributions of files must retain the above copyright notice.
  *
- * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
- * @link          https://cakephp.org CakePHP(tm) Project
- * @since         3.0.0
- * @license       https://opensource.org/licenses/mit-license.php MIT License
+ * @copyright    Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
```

Comment:
These remain :)

Reply:
arughhhhh

Reply:
Is there an automated way to see this??

Reply:
I have made changes as per the PR link. That was helpful.